src/lib/stripe/price.ts

```diff
@@ -0,0 +1,15 @@
+import Stripe from 'stripe'
+
+if (!process.env.STRIPE_SECRET_KEY) {
```

Comment:
this is a concern with this because technically it could be imported into a frontend file and exposed, where currently it's being used in a SSR method, and doesn't expose it.

Alternative would be to be more explicit and migrate to a `/src/pages/api` function where it might be more clear it's not meant to be used in the UI. 🤔